### PR TITLE
DB845c warrior WiFi and Audio

### PR DESCRIPTION
**Pull request by alimon**

### CODE REVIEW

File: conf/machine/dragonboard-845c.conf [hunk 1]
```diff
@@ -16,6 +16,8 @@ MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS += " \
     kernel-modules \
     ${@bb.utils.contains('DISTRO_FEATURES', 'bluetooth', 'linux-firmware-qca', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'opengl', 'mesa-driver-msm', '', d)} \
+    pd-mapper \
```
Comment (ndechesne) [inline conf/machine/dragonboard-845c.conf]: why not in MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS in sdm845.inc like we do for qrtr and rmtfs. any reason why it should be handled differently?

File: recipes-bsp/firmware/firmware-qcom-dragonboard845c_20190529180356-v4.bb [hunk 2]
```diff
@@ -3,9 +3,13 @@ DESCRIPTION = "QCOM Firmware for DragonBoard 845c"
 LICENSE = "Proprietary"
 LIC_FILES_CHKSUM = "file://LICENSE.qcom.txt;md5=cbbe399f2c983ad51768f4561587f000"
 
-SRC_URI = "https://releases.linaro.org/96boards/dragonboard845c/qualcomm/firmware/RB3_firmware_${PV}.zip"
+SRC_URI = " \
+	https://releases.linaro.org/96boards/dragonboard845c/qualcomm/firmware/RB3_firmware_${PV}.zip \
+	git://github.com/andersson/linux-firmware.git;branch=db845c-wifi;name=andersson;destsuffix=git \
```
Comment (ndechesne) [inline recipes-bsp/firmware/firmware-qcom-dragonboard845c_20190529180356-v4.bb]: i believe the files in Bjorn's PR are coming from the RB3_firmware package. they might have been post processed, but they are from the same package. we should use only the RB3_firmware package.

**Comment by ndechesne** [comment]
I suppose that it supersedes #122 , if so, please close it.

File: conf/machine/dragonboard-845c.conf (hunk 1 above)
Comment (alimon) [reply conf/machine/dragonboard-845c.conf]: Ok, I will update it.

File: recipes-bsp/firmware/firmware-qcom-dragonboard845c_20190529180356-v4.bb (hunk 2 above)
Comment (alimon) [reply recipes-bsp/firmware/firmware-qcom-dragonboard845c_20190529180356-v4.bb]: Most of the files comes from RB3_firmware package, only boad-2.bin was built by Bjorn based on RB3_firmware package.

Comment (alimon) [reply recipes-bsp/firmware/firmware-qcom-dragonboard845c_20190529180356-v4.bb]: @andersson Any other details that I miss?

**Pull request merged**